glotaran/project/result.py

```diff
@@ -310,6 +310,39 @@ def verify(self) -> bool:
 
         return True
 
+    def create_clp_guide_dataset(self, dataset: str, clp_label: str) -> xr.Dataset:
```

Comment:
Please add an optional argument `overwrite_cpl_value: float | None = None` in order to generate backwards compatible `ascii` which can be used with `TMP` and `TIMP`.

Reply:
As discussed with @joernweissenborn we will just take the first value of the `midel_dimension`

Reply:
Fixed in 394140